### ISSUE TITLE
Your IOReg report

### ISSUE THREAD

**Issue by fabioavelino** [jbwharris/hackintosh-msi-GL72M-7RDX#1]
Hi,

I'm trying to make my DW1820A work on my laptop and would like to see your IOReg to see in which slot it's connected on your laptop the wifi card and trying to replicate your ACPI config in my computer.

Is it possible to get your IOReg so ? Thanks !

**Comment by jbwharris** [comment]
Sure, though my laptop is at home, so I probably won't be able to send anything until tonight. This card does work for WiFi, though it's been giving me a lot of grief getting the Bluetooth to work.

**Comment by jbwharris** [comment]
Here's the IOReg

[ioreg-output.ioreg.zip](https://github.com/jbwharris/hackintosh-msi-GL72M-7RDX/files/2843356/ioreg-output.ioreg.zip)

**Comment by fabioavelino** [comment]
Thanks man ! Very much appreciated ! It will help me for sure

**Comment by jbwharris** [comment]
I was able to get the Bluetooth working, though my build is having some issues at the moment that I'm trying to resolve. My steps were pretty much disable SIP, use Hackintool to properly map my USB ports and using BrcmFirmwareRepo.kext and BrcmPatchRAM2.kext. Now it works pretty well, been able to connect to speakers, use Handoff etc.

**Comment by jbwharris** [comment]
Just wanted to check in on whether you made any progress on the Bluetooth @fabioavelino? I just pushed a new Mojave build and was able to get fully functional Bluetooth going. It was giving me issues previously, I ended up implementing a lot of the edits [from here](https://github.com/RehabMan/OS-X-Clover-Laptop-Config/blob/master/config_HD615_620_630_640_650.plist) to fix my HDMI and it ended up fixing my Bluetooth as well.
